Sources/Plasma/FeatureLib/pfAnimation/plRandomCommandMod.cpp

```diff
@@ -297,5 +297,5 @@ void plRandomCommandMod::IRetry(float secs)
 
     plAnimCmdMsg* msg = new plAnimCmdMsg(nil, GetKey(), &t);
     msg->SetCmd(plAnimCmdMsg::kContinue);
-    plgDispatch::MsgSend(msg);
+    msg->Send();
```

Comment:
I feel like this kind of change should be in its own PR, as this is not a trivial replacement...

Reply:
How so? The body of `plMessage::Send()` is basically just a call to `plgDispatch::MsgSend()`...

Reply:
Oh, so it is...  I was looking from the other direction (at `plgDispatch::MsgSend`, assuming it called `msg->Send()` somewhere)...   >.>